### ISSUE TITLE
Name sheets in export

### ISSUE THREAD

**Issue by rap2hpoutre** [rap2hpoutre/fast-excel#40]
According to [this comment](https://github.com/rap2hpoutre/fast-excel/pull/24#issuecomment-397243843), it would be great to be able to name sheets. As suggested by @Harty, it could be done in `SheetCollection` constructor:

```php
$sheets = new SheetCollection(
    [User::all(), Project::all()], 
    ['users sheet', 'project sheet']
);
```

So it means a new property (`names`) could be added to `SheetCollection` then it could be passed as a second argument in the `SheetCollection` constructor.

The main advantage to add it in the constructor is that (in a second step), it could be used in import process too: the sheet names would be loaded from the `importSheets` function that returns the `SheetCollection`. So that's cool.

One disadvantage of creating the name of the sheets via `SheetCollection` constructor is that there is no way to name a sheet when exporting only a `Collection` (e.g only one sheet) but I'm not sure it's an actual issue: the main goal is to **name sheets in multiple sheets export**.

**Comment by SimoTod** [comment]
How about 
```$sheets = new SheetCollection([
    'users sheet' => User::all(),
    'project sheet' => Project::all()
]);``` and still supporting `$sheets = new SheetCollection([User::all(), Project::all()]);` if you don't want to change sheet names?

When exporting a single collection, you would use `$sheets = new SheetCollection(['users' => User::all()]);`

**Comment by rap2hpoutre** [comment]
@SimoTod ... 👍 👍

You are totally right. Your suggestion is really good, thank you so much! I will develop it following your advice. So:

```
$sheets = new SheetCollection(['users' => User::all()]);
(new FastExcel($sheets))->export('file.xlsx');
```

**Comment by rap2hpoutre** [comment]
Fixed via https://github.com/rap2hpoutre/fast-excel/commit/059442e4e124fab5999fd4d64a19c0a6626a33f0 available in v0.9.0
